Main.cs

```diff
@@ -612,6 +628,7 @@ public static void Run(string[] _args)
                             }
                         }
                     }
+                    return;

```

Comment:
I want to rework all commands to <code>enum</code>s in any case. It makes sense to store help info and other such stuff near the classes. IMO it could be in winsw-2.0 scope.

Enums in C# are not so powerful compared to Java ones, but helper classes will resolve this issue.